lib/qa/authorities/getty/aat.rb

```diff
@@ -23,7 +29,7 @@ def sparql(q)
                  skos:inScheme <http://vocab.getty.edu/aat/> ;
                  gvp:prefLabelGVP [skosxl:literalForm ?name].
               FILTER regex(?name, \"#{search}\", \"i\") .
-            } LIMIT 10"
```

Comment:
To allow for more results. Still seems very quick, AAT search for art c. 600ms. Happy to put the limit back. TGN returns more results but still reasonably performant on queries that aren't too broad.